### PR TITLE
feat(organizationTeams): added organization teams endpoints

### PR DESCRIPTION
**Pull request by vbeleuta**
Added /organization/orgId/teams endpoints: createTeam, getTeam (with update and delete), getTeams.

### CODE REVIEW

File: test/integration/team-integration.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  })
+
+  t.test('Gets team', (t) => {
+    t.plan(4)
+    return organization.getTeam('7pIEx2fMx53SSR1jd7C46M')
+      .then((response) => {
+        t.ok(response.sys, 'sys')
+        t.ok(response.sys.id, '7pIEx2fMx53SSR1jd7C46M')
+        t.ok(response.sys.type, 'Team')
+        t.ok(response.name, 'SDK test team [DO NOT DELETE]')
```
Comment (guilebarbosa) [inline test/integration/team-integration.js:19]: Should you be using `t.equals` here?

**Comment by vbeleuta** [comment]
@matthew-contentful looks like the create space integration tests keep failing without change, did the plan limit get reached or the planId is wrong (the error message: SubscriptionManagement::CreateSpaceRatePlan::InvalidProductRatePlanId)? do you have any insight? thanks!

**Comment by matthew-contentful** [comment]
> @matthew-contentful looks like the create space integration tests keep failing without change, did the plan limit get reached or the planId is wrong (the error message: SubscriptionManagement::CreateSpaceRatePlan::InvalidProductRatePlanId)? do you have any insight? thanks!

No insight, just that the error is a new (within the last 10 days?) one. Perhaps a plan has ceased to exist?

**Comment by vbeleuta** [comment]
> > @matthew-contentful looks like the create space integration tests keep failing without change, did the plan limit get reached or the planId is wrong (the error message: SubscriptionManagement::CreateSpaceRatePlan::InvalidProductRatePlanId)? do you have any insight? thanks!
> 
> No insight, just that the error is a new (within the last 10 days?) one. Perhaps a plan has ceased to exist?

hmm, yesterday the tests passed so looks like a very recent change

**Pull request merged**